src/core/infrastructure/entities/Tribe.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, Unique, OneToMany, PrimaryGeneratedColumn } from "typeorm";
+import { UserRecord } from "./User";
+
+@Entity({ name: "tribes" })
+@Unique(["id"])
```

Comment:
Don't need to specify this because of `@PrimaryGeneratedColumn("uuid", { name: "id" })`